lib/fuzzily/searchable.rb

```diff
@@ -36,20 +38,22 @@ def fuzzily_searchable(*fields)
 
       def _find_by_fuzzy(_o, pattern, options={})
         trigrams = _o.trigram_class_name.constantize.
+          limit(options.fetch(:limit, 10)).
           offset(options.fetch(:offset, 0)).
           for_model(self.name).
           for_field(_o.field.to_s).
+          within(true ? self.pluck(:id) : NIL). # TODO on searching without an where this is not desireable
```

Comment:
Filter `owner_id`'s before searching in the trigrams in such we can filter and offset without a problem.